src/tiledb/cloud/bioimg/exportation.py

```diff
@@ -20,14 +61,17 @@ def export(
     num_batches: Optional[int] = None,
     resources: Optional[Mapping[str, Any]] = None,
     compute: bool = True,
+    local: bool = False,
     namespace: Optional[str] = None,
+    verbose: bool = False,
+    output_ext: str = "tiff",
```

Comment:
We use as a default value in the UDF the `Converters.OMETIFF`, so I thought it would make sense.

```python
to_bioimg(
    input,
    output,
    converter=Converters.OMETIFF,
    config=src_cfg,
    output_config=dest_cfg,
    verbose=verbose,
    **kwargs,
    )
```